kong/plugins/tcp-log/handler.lua

```diff
@@ -27,6 +27,15 @@ local function log(premature, conf, message)
     return
   end
 
+  if conf.tls then
+    ok, err = sock:sslhandshake(true, conf.tls_sni, false)
+    if not ok then
+      ngx.log(ngx.ERR, "[tcp-log] failed to perform TLS handshake to ",
+              host, ":", port, ": ", err)
```

Comment:
style: we typically apply an exception to `ngx.log` and align subsequent arguments with the 2nd one (`"...`) 😅